webCommander/centralHub/templates/tweetFramework/x_handle_submission_form.html

```diff
@@ -9,7 +9,15 @@ <h1>Submit Text</h1>
     <form method="post">
         {% csrf_token %}
         {{ form_handle.as_p }}
+        {% if messages %}
```

Comment:
this should go in the base.html template so it's central to all child templates that inherit from it